.travis.yml

```diff
@@ -1,69 +1,124 @@
 language: python
-python:
-- '3.6'
-- '3.7-dev'
 
-git:
-  depth: 200
-sudo: required
-dist: trusty
+matrix:
+  include:
+  - os: linux
+    dist: trusty
+    python: 3.7
+    env:
+      - REPO=https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+      - OS=linux-64
+  - os: linux
+    dist: trusty
+    python: 3.6
+    env:
+      - REPO=https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+      - OS=linux-64
+  - os: linux
+    dist: xenial
+    python: 3.7
+    env:
+      - REPO=https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+      - OS=linux-64
+  - os: linux
+    dist: xenial
+    python: 3.6
+    env:
+      - REPO=https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+      - OS=linux-64
+  - os: osx
+    language: generic
+    env:
+    - REPO=https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
+    - TRAVIS_PYTHON_VERSION=3.7
+    - OS=osx-64
+  - os: osx
+    language: generic
+    env:
+    - REPO=https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
+    - TRAVIS_PYTHON_VERSION=3.6
+    - OS=osx-64
+
 before_install:
   - sudo apt-get update
   - ldd --version
   - gcc --version
   - export START=$(pwd)
+
 install:
-- if [[ "$TRAVIS_PYTHON_VERSION" == "3.7-dev" ]]; then export VADD="py37"; else export VADD="py36";  fi
-- wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
-  -O miniconda.sh
-- bash miniconda.sh -b -p $HOME/miniconda
-- export PATH="$HOME/miniconda/bin:$PATH"
-- hash -r
-- conda config --set always_yes yes --set changeps1 no
-- conda install conda-build
-- conda update -q conda
-- conda update -n root conda-build
-- conda config --set anaconda_upload no
-- conda config --append channels conda-forge
-- conda config --append channels tofuproject
-- conda info -a
-- if [[ "$TRAVIS_PYTHON_VERSION" == "3.7-dev" ]]; then export THIS_PY_VERSION="3.7";
-  else THIS_PY_VERSION=$TRAVIS_PYTHON_VERSION;
-  fi
-- conda install -q python=$THIS_PY_VERSION conda-build anaconda-client nose
-  nose-timer coverage codecov
-- export REV=$(python -c "import _updateversion as up; out=up.updateversion(); print(out)")
-- export VERSION=$(echo $REV | tr - .)
-- echo $REV
-- conda build conda_recipe
-- export PKG_DIR=$HOME/miniconda/conda-bld/linux-64/
-- conda install tofu --use-local
+  - if [[ "$TRAVIS_PYTHON_VERSION" == "3.7" ]]; then export VADD="py37";
+    else export VADD="py36";  fi
+  - wget "$REPO" -O miniconda.sh
```

Comment:
Change here since we created an environmental variable for the source of the conda repo